### DISCUSSION TITLE
Implementing additional features extraction using timm.

### DISCUSSION THREAD

**Discussion by AgenoDrei** [v-iashin/video_features#135] [Ideas]
I tested using the Pytorch Image Models (timm) library to extract RETFound and Dino v2 features for my own research project. The timm library makes this pretty simple by giving a unified format for models and their configuration.

Would you be interested in dino v2 features using this library for your own project? I could start working on a fork in a few weeks.

**Comment by v-iashin** [comment]
Hi. We support extraction with timm models.

https://v-iashin.github.io/video_features/models/timm/

Do you mean that something is not working?

**Reply by AgenoDrei** [reply]
Hey, I did miss that in the documentation. Nevermind and thanks for providing the great library! :)
